azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/config/AzureComputeServiceContextModule.java

```diff
@@ -152,10 +145,6 @@ public String azureImagePublishers() {
          return azureImagePublishersProperty;
       }
 
-      public String azureDefaultImageLogin() {
-         return azureDefaultImageLoginProperty;
-      }
-
```

Comment:
We use the existing jclouds properties, so there is no need for these custom properties anymore.